util/otbnsim/README.md

```diff
@@ -0,0 +1,97 @@
+# OpenTitan Big Number Python Model
+
+## Generate documentation
+
+```console
+$ python -m otbnsim.doc
+++++ Instruction Formats
+{'id': 'L',
+ 'fields': [{'name': 'bodysize',
+             'base': 20,
+             'size': 12,
+             'description': '',
+             'static': False,
+             'value': None},
+            {'name': 'funct3',
+             'base': 12,
+             'size': 3,
+             'description': '',
+             'static': True,
+             'value': None},
+
+[...]
+
+++++ Instructions
+{'format': 'L',
+ 'description': 'Loop (indirect)\n'
+                '\n'
+                'Repeat a sequence of code multiple times. The number of '
+                'iterations is a GPR\n'
+                'value. The length of the loop is given as immediate.\n'
+                '\n'
+                'Alternative assembly notation: The size of the loop body is '
+                'given by the\n'
+                'number of instructions in the parentheses.\n'
+                '\n'
+                'LOOP <grs> (\n'
+                '  # loop body\n'
+                ')',
+ 'asm_signature': 'loop <iter>, <bodysize>',
+ 'code': '        model.state.loop_start(int(reg[rs1]), int(bodysize))'}
+
+[...]
+```
+
+## Assembler
+
+Assemble to verify everything works:
+
+```console
+$ otbn-asm << EOF
+> LOOPI 8 (
+>   addi x2, x2, 1
+> )
+> EOF
+loopi 8, 1
+addi x2, x2, 1
+```
+
+You can also use an assembly file obviously:
+
+```console
+$ otbn-asm test.S
+```
```

Comment:
I'm not sure whether you need this. You might hope that a reader of this README understands what a here-doc is... :-/